app/client/css/main.styl

```diff
@@ -41,15 +18,22 @@ html
   background-color: $color-white-soft
   color: $color-black-soft
 
+
 // TYPOGRAPHY
 
 $font-size-basic = 19px
+$font-size-mobile = 16px
+
 $font-leading-basic = 1.4em
 $font-leading-btn = 2.2em
 
-$font-size-h1 = 40px
-$font-size-h2 = 30px
-$font-size-h3 = 24px
+$font-size-h1 = $font-size-basic * 2.105263158
```

Comment:
Turned those absolutes into multipliers so I could keep the same proportions for the smaller mobile headings